config/samples/demo/bfdprofile.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: metallb.io.metallb.io/v1alpha1
+kind: BFDProfile
+metadata:
+  name: bfdprofilefull
+  namespace: metallb-system
+spec:
+  receiveInterval: 35
+  transmitInterval: 35
+  detectMultiplier: 37
+  echoReceiveInterval: 38
```

Comment:
as the echoReceiveInterval is defined as a string, this YAML gives me an error for an invalid value

Reply:
Thanks! I am changing it to be an int and to be single echoInterval, to reflect the metallb api.